### PR TITLE
CHANGELOG: update for LeaseTimeToLive returns error if leader changed

### PR DESCRIPTION
**Pull request by fuweid**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.4.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 ## v3.4.32 (TBD)
 
+### etcd server
+- Fix [*: LeaseTimeToLive returns error if leader changed](https://github.com/etcd-io/etcd/pull/17705).
```
Comment (serathius) [inline CHANGELOG/CHANGELOG-3.4.md:10]:
```suggestion
- Fix [LeaseTimeToLive returns error if leader changed](https://github.com/etcd-io/etcd/pull/17705).
```

File: CHANGELOG/CHANGELOG-3.5.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.14 (TBD)
 
+### etcd server
+- Fix [*: LeaseTimeToLive returns error if leader changed](https://github.com/etcd-io/etcd/pull/17704).
```
Comment (serathius) [inline CHANGELOG/CHANGELOG-3.5.md:10]:
```suggestion
- Fix [LeaseTimeToLive returns error if leader changed](https://github.com/etcd-io/etcd/pull/17704).
```

File: CHANGELOG/CHANGELOG-3.4.md (hunk 1 above)
Comment (serathius) [reply CHANGELOG/CHANGELOG-3.4.md:10]: Doesn't need to be name of PR 1to1.

**Comment by fuweid** [comment]
ping @serathius @ahrtr

**Review by ahrtr — APPROVED** [review]

**Pull request merged**